step-templates/remove-server-from-azure-load-balancer.json

```diff
@@ -0,0 +1,57 @@
+{
+  "Id": "5658d525-2a04-47da-85a0-00244976d811",
+  "Name": "Remove Server From Azure Load Balancer",
+  "Description": "Remove Server From Azure Load Balancer",
```

Comment:
Same here, as this step template requires the use of a Service Principal, could you mention that in the description?